### DISCUSSION TITLE
Welcome to spaceduck Discussions!

### DISCUSSION THREAD

**Discussion by pineapplegiant** [pineapplegiant/spacedirt-theme#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jacobranz** [comment]
Hello there, I have recently implemented this solution on one of my networks and am a big fan. Its very clean and a nice way to customize vim in a clean way! I did have one question though, and I am sure it is a simple fix. When I go to right click paste in insert mode, it opens visual mode and does not paste. Is there a better alternative to pasting? Or am I just missing a setting that needs to be there?
